server/controllers/ver.js

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
```

Comment:
oh wait, just saw your [other comment](https://github.com/mozilla/chronicle/pull/325#issuecomment-74117037) about this, it's fine